packages/nextjs-mf/src/default-delegate.ts

```diff
@@ -151,7 +173,10 @@ const loadScript = (keyOrRuntimeRemoteItem: string | RuntimeRemote) => {
 
```

Comment:
The exported Promise does not handle possible rejections from 'importDelegatedModule'. This could lead to unhandled Promise rejections.